src/filesystem/implementations/local.h

```diff
@@ -60,12 +60,44 @@ class LocalFileSystem : public FileSystem {
   Status MakeTemporaryDirectory(
       std::string dir_path, std::string* temp_dir) override;
   Status DeletePath(const std::string& path) override;
+
+ private:
+  inline std::string GetOSValidPath(const std::string& path);
+  static constexpr const char* kWindowsLongPathPrefix = "\\\\?\\";
```

Comment:
Why not move this into GetOSValidPath as static local variable

Reply:
I can do that.